### PR TITLE
Mass notification table

### PR DESCRIPTION
**Pull request by fseipel**

### CODE REVIEW

File: Resources/Private/Partials/Backend/Application/ListNotification.html [hunk 1]
```diff
@@ -2,9 +2,15 @@
...
 
-<f:form action="newMassNotification">
+<div class="panel panel-default">
+	<div class="panel-body">
+		<f:form.button class="newApplications btn btn-default" /><f:translate key="tx_ats.application.status.10"/></button>
+	</div>
+</div>
 
-	<table class="table">
+<f:form action="newMassNotification" onsubmit="$('.applications-list').dataTable().fnDestroy();">
```
Comment (mbrodala) [inline Resources/Private/Partials/Backend/Application/ListNotification.html]: Please move this to the regular script file.

File: Resources/Public/JavaScript/ApplicationsModule.js [hunk 2]
```diff
@@ -77,6 +77,11 @@ require(
 			});
 		});
 
+        // Destroy the dataTable on submit to keep the right order in the BFCache
+        $('#newMassNotification').submit(function(){
```
Comment (mbrodala) [inline Resources/Public/JavaScript/ApplicationsModule.js]: I'd prefer `.on('submit', function() ...` instead here.

File: Resources/Public/JavaScript/ApplicationsModule.js [hunk 3]
```diff
@@ -77,6 +77,11 @@ require(
...
+            $('.applications-list').dataTable().fnDestroy();
```
Comment (mbrodala) [inline Resources/Public/JavaScript/ApplicationsModule.js]: Please use `$(this).find('.applications-list')...` instead here, this avoids unexpected side-effects.

**Comment by pm-sschreiber** [comment]
@fseipel the button to select all new applications is very misleading, it looks like the button to create a new application (probably uses the same label). DE:

image

The button should be labelled as "Select all new applications" / "Alle neuen Bewerbungen auswählen".

**Pull request merged**